x-pack/plugin/eql/qa/common/licenses/jtoml-NOTICE

```diff
@@ -0,0 +1,2 @@
+jtoml
+TOML Parser for Java
```

Comment:
@costin I was not sure what to put in here, but now looks like I need this files for jtoml dependency verification task in EQL QA project.
let me know.

Reply:
The build requires  a file, at least an empty one :-) Can remove the content.